nano/slow_test/node.cpp

```
@@ -443,12 +443,13 @@ TEST (store, pruned_load)
 					nano::block_hash random_hash;
 					nano::random_pool::generate_block (random_hash.bytes.data (), random_hash.bytes.size ());
 					store->pruned.put (transaction, random_hash);
+					hashes.insert (random_hash);
 				}
 			}
 			if (!nano::rocksdb_config::using_rocksdb_in_tests ())
 			{
 				auto transaction (store->tx_begin_write ());
-				for (auto k (0); k < batch_size / 2; ++k)
+				for (auto k (0); !hashes.empty () && k < batch_size / 2; ++k)
```

Do we need the check `!hashes.empty ()`?
This is a unit test, if anything is wrong, we want to fail loud and clear rather than handle it quietly.
So I would say that we should ASSERT that hashes is not empty.